torch/distributed/fsdp/api.py

```diff
@@ -46,12 +46,22 @@ class ShardingStrategy(Enum):
       :class:`DistributedDataParallel` API. For gradients, this strategy
       synchronizes them (via all-reduce) after the backward computation. The
       unsharded optimizer states are updated locally per rank.
+    - ``HYBRID_SHARD``: Apply ``FULL_SHARD`` within a node, and data parallelism across
```

Comment:
```suggestion
    - ``HYBRID_SHARD``: Apply ``FULL_SHARD`` within a node and replicate parameters across
```

`FULL_SHARD` is still implementing data parallelism; only it is _sharded_ data parallelism.

Reply:
[Easy] I recommend changing before landing